lighthouse-core/test/computed/metrics/cumulative-layout-shift-test.js

```diff
@@ -298,7 +298,7 @@ describe('Metrics: CLS', () => {
 
       it('ignores layout shift data from other tabs', async () => {
         const trace = createTestTrace({timeOrigin: 0, traceEnd: 2000});
-        const mainFrame = trace.traceEvents[0].args.frame;
+        const mainFrame = trace.traceEvents.find(e => e.name === 'navigationStart').args.frame;
```

Comment:
these `mainFrame` changes are a little awkward (and one reason why we should consider an ease of use refactor at some point), but still straightforward. `navStart` isn't necessarily the first event.